v3/generator/schemas/schemas.go

```diff
@@ -203,16 +204,32 @@ func renderSchemaInternal(schemaName string, s *base.Schema, output *bytes.Buffe
 }
 
 func renderSimpleTypeEnum(typeName string, s *base.Schema) string {
+	for _, e := range s.Enum {
+		if s.Minimum != nil || s.Maximum != nil {
+			return ""
+		}
+
```

Comment:
When the new public-api with the sanitized spec is published, you may want to delete these lines again and regenerate before merging.